sale_order_type/views/account_invoice_view.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
```

Comment:
remove data tag as is is indicated in this page: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0